tasks/main.yml

```diff
@@ -30,3 +30,7 @@
     state=stopped
     enabled=no
   when: not ntp_enabled
+
+- name: Setup ntp.conf file and Restart NTP
```

Comment:
How about "Generate ntp.conf." for the name here (note the period at the end)?

This line doesn't actually restart NTP, so it doesn't need to be mentioned here.